app/src/main/java/pl/futurecollars/invoicing/controller/InvoiceController.java

```diff
@@ -0,0 +1,51 @@
+package pl.futurecollars.invoicing.controller;
+
+import java.util.List;
+import java.util.Optional;
+import lombok.Generated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pl.futurecollars.invoicing.db.InMemoryDataBase;
+import pl.futurecollars.invoicing.model.Invoice;
+import pl.futurecollars.invoicing.service.InvoiceService;
+
+@Generated
+@RestController
+public class InvoiceController {
+
+  private final InvoiceService invoiceService = new InvoiceService(new InMemoryDataBase());
+
+  @GetMapping("/hello")
+  public String helloWorld() {
+    return "Hello Man";
+  }
+
+  @PostMapping("/add-invoice")
```

Comment:
Dużo lepszym pomysłem jest zrobienie ścieżki w stylu /invoice, bezpośrednio w nazwie zasobu nie używamy czasowników, to metoda HTTP definiuje to co możemy na takiej ścieżce uzyskać. Pamiętaj, że możesz mieć kilka endpointów pod jedną ścieżką dopóki rożnią się metodami HTTP. Poczytaj sobie o nazywaniu zasobów w restowych serwisach np tutaj: https://restfulapi.net/resource-naming/